### PR TITLE
Add @datauy from Uruguay

### PR DESCRIPTION
**Pull request by gabelula**
Our website www.datauy.org

### CODE REVIEW

File: CNAME [hunk 1]
```diff
@@ -1 +1 @@
-government.github.com
+gobierno.datauy.org
```
Comment (nschonni) [inline CNAME:1]: Uhh...

Comment (gabelula) [inline CNAME:1]: oops. I did not realize I was including it in the pull request. Sorry.

Comment (nschonni) [inline CNAME:1]: Yeah, that's why it's good to use feature branches :wink:

**Comment by gabelula** [comment]
I close this one and I make a new pull request with the right commit.

**Pull request closed without merging**